src/app/features/shared/components/molecules/Table/Table.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react"
+import { Button } from "@datapunt/asc-ui"
+import { ChevronRight } from "@datapunt/asc-assets/lib"
+import { withKnobs, boolean } from "@storybook/addon-knobs"
+
+import Table from "./Table"
+
+export default {
+  title: "Shared/Molecules/Table",
+  decorators: [withKnobs]
+}
+
+const columns = [
+  "ID",
+  "First name",
+  "Surname",
+  "Email",
+  "Gender",
+  "Ip",
+  <>&nbsp;</>
```

Comment:
Maybe we can use `undefined` or something more abstract to allow for columns without a header? This seems like a hack to me at first sight.

Reply:
Yeah, that might be better